charts/newrelic-pixie/values.yaml

```diff
@@ -0,0 +1,28 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+# cluster: ""
+
+# The New Relic license key
+# licenseKey: ""
+
+# The Pixie API key
+# apiKey: ""
+
+# If you already have a secret where the New Relic license key and the Pixie API key are used, indicate its name here
+# customSecretName:
+# The key in the custom secret that contains the New Relic license key
+# customSecretLicenseKey:
+# The key in the custom secret that contains the Pixie API key
+# customSecretApiKeyKey:
+
+image:
+  repository: newrelic/newrelic-pixie-integration
+  tag: latest
+  pullPolicy: Always
+
+resources:
+  limits:
+    memory: 80M
+  requests:
+    cpu: 100m
+    memory: 30M
```

Comment:
We usually don't have default resources in the charts (specially not `requests`), as they might prevent installation on some environments.

Reply:
We have them defined for the `newrelic-infrastructure` chart. It's better that it's not installed than not running properly due to lack of resources.

Reply:
You are right. I think we do no have them on nri-prometheus but I'm not sure about the reason. Let's go with them :)